recipes/pmda/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "pmda" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 90ffbfa4e4ac6617e7811cd57ada57687daa654fbb176b835aec0f8b3d38f85c
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - dask
+    - mdanalysis>=0.18.0
```

Comment:
@kain88-de I think we can just use 0.19 now?  The package uses the develop branch, but I think that's solved now

Reply:
Use >= 0.19.0